docs/source/examplecleanup.rst

```diff
@@ -111,9 +137,9 @@ An End-to-End example walkthrough of a tartufo scan and the process of purging t
       git push
 
 
-#. Danger Will Robinson, Danger! 
+#. Danger Will Robinson, Danger!
```

Comment:
Probably want to add a note that anyone that has cloned the repo prior to this should re-clone to prevent reintroducing the bad history

Reply:
@mxhenry-godaddy Good point! Do you think this will suffice? https://github.com/godaddy/tartufo/pull/111/files#diff-cd26aa27595802087598d66aa01eac4dR128-R135

Visible here at point number 7: https://tartufo.readthedocs.io/en/v2-docs/examplecleanup.html

Reply:
yep